4.0/init.sh

```diff
@@ -81,6 +81,14 @@ sudo -u nominatim nominatim admin --check-database
 
 if [ "$REPLICATION_URL" != "" ]; then
   sudo -E -u nominatim nominatim replication --init
+  if [ "$FREEZE" = "true" ]; then
+    echo "Skipping freeze because REPLICATION_URL is not empty"
+  fi
+else
+  if [ "$FREEZE" = "true" ]; then
+    echo "Freezing database"
+    sudo -u nominatim nominatim freeze
```

Comment:
This block is going to be started on every container start up. Is it problematic to call several times?

Reply:
I reviewed the [Nominatim code that freezes the database](https://github.com/osm-search/Nominatim/blob/85d65a2fd27cc5bc47685e7759a1895e1f9f2236/nominatim/tools/freeze.py) and it shouldn't be an issue. The `freeze` command drops some tables and deletes the flatnode file only if they exist, and they won't on subsequent runs.